### DISCUSSION TITLE
Right modifier keys are invalid keybinds

### DISCUSSION THREAD

**Discussion by rhodes-b** [ghostty-org/ghostty#4236] [Feature Requests, Ideas] [answered]
https://ghostty.org/docs/config/reference#keybind
does not list any of the right modifier keys as valid keybinds even though they exist in the keymap

https://github.com/ghostty-org/ghostty/blob/d6e76858164d52cff460fedc61ddf2e560912d71/src/input/key.zig#L399-L402

This will give an error on parsing if you add them to your config

The terminal inspector also shows these as 2 key presses (that may be correct behavior)
![image](https://github.com/user-attachments/assets/2af0d152-e986-4a63-8561-679826fba05f)

**Comment by rhodes-b** [comment, marked as answer]
Closing as dupe since i have seen mitchell talk about recognizing left modifiers before (so i assume it also applys to right modifiers)

**Reply by 00-kat** [reply]
#3364 is the one.
